lua/autorun/sh_netstream.lua

```diff
@@ -0,0 +1,164 @@
+--[[
```

Comment:
you don't need this shit or sh_pon, use built in networking

Reply:
Okie. sh_pon and netstream just makes code looks more compact, but as you wish.